### ISSUE TITLE
Präzisere Vorgabe zur Sequenzlänge bei Petrinetz-Folgen

### ISSUE THREAD

**Issue by jvoigtlaender** [fmidue/modelling-tasks#289]
![image](https://github.com/user-attachments/assets/e242d36c-34aa-4900-a4d0-c316df8bb0d3)

Wenn bekannt und vorgegeben ist, dass keine Lösung mit weniger als 7 Transitionen existiert, dann könnte statt
> als maximal 7-elementige Auflistung

auch gleich
> als genau 7-elementige Auflistung

verlangt werden.

**Comment by jvoigtlaender** [comment]
Wobei dann noch die Frage nach Fairness aufkommen könnte, wenn trotzdem auch 5-elementige Folgen angenommen (also nicht syntaktisch zurückgewiesen) werden und sich damit Teilpunkte erreichen lassen (manche Studierende sich aber nicht "trauen", so etwas abzugeben, weil "genau 7" da stand).

**Comment by marcellussiegburg** [comment]
Dem könnte entgegengewirkt werden, indem das Bewertungsschema vorab kommuniziert wird. Das wäre aus meiner Sicht prinzipiell für alle Aufgaben sinnvoll (wegen Transparenz und Fairness). [Der Formulierungsvorschlag suggeriert, dass bei weniger als 7 Transitionen syntaktisch abgelehnt werden sollte. Ob das dann auch gemacht werden sollte, bleibt sicherlich noch immer streitbar. Bei dieser Aufgabe konkret wäre auch #246 relevant.]
Dieses Issue weiter sprengend, wären zu diesem Punkt auch noch andere Überlegungen zu treffen:
- Sollte das Anzeigen des Bewertungsschemas von der Aufgabenstellung separiert werden (z.B. zum Verstecken, wenn gar keine Punkte verteilt werden)?
- Wie bleibt das möglichst mit Bewertungsanpassungen synchronisiert?

**Comment by jvoigtlaender** [comment]
Der konkrete oben öffnende Kommentar dürfte durch #369, #256 weitgehend obsolet werden?

**Comment by marcellussiegburg** [comment]
Ja, das sollte es weitestgehend beheben. Die Betrachtungen zu Fairness und Transparenz (https://github.com/jvoigtlaender/modelling-tasks-internal/issues/289#issuecomment-2520235530) sind wohl ein separates Issue, dass vielleicht generellerer Überlegungen bedarf (sowohl zum didaktischen Bedarf als auch zur technischen Umsetzung; vielleicht auch (zunächst) Autotool-seitig)?
